content/docs/hooks-reference.md

```diff
@@ -98,7 +98,10 @@ const [state, setState] = useState(() => {
 
 Se você atualizar o estado do Hook com o mesmo valor do estado atual, React irá pular a atualização sem renderizar os filhos ou disparar os efeitos. (React usa o algoritmo de comparação [`Object.is`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is#Description).)
 
-Note that React may still need to render that specific component again before bailing out. That shouldn't be a concern because React won't unnecessarily go "deeper" into the tree. If you're doing expensive calculations while rendering, you can optimize them with `useMemo`.
+Note que o React ainda pode precisar renderizar aquele componente específico novamente antes de pular para fora da ação. Isso não deve ser um problema porque o React não vai se aprofundar desnecessariamente na árvore. Se você está fazendo cálculos de alto custo enquanto renderiza, você pode optimizá-los usando o useMemo.
+
+
+Note que o React pode ainda precisar renderizar esse componente específico novamente antes de sair. Isso não deveria ser uma preocupação porque o React não irá ser "mais profundo" do que o necessário na árvore. Se você está fazendo um processamento mais demorado enquanto renderizada, você pode otimizar isso usando `useMemo`.
```

Comment:
Estou falando desse trecho aqui, é o mesmo tento que vc duplico a tradução no mesmo arquivo, consegue remover uma delas, e sim na versão publica ainda não vai aparecer a tradução, apenas depois que seu PR for dado merge

Reply:
Só, agora entendi.
Feito o ajuste.